articles/access_controls.md

```diff
@@ -1,58 +1,57 @@
 ---
-title: "Access and Content Restrictions"
+title: "Access Controls and Data Governance: Sharing Settings and Conditions for Use"
 layout: article
-excerpt: Learn how to share and restrict content on Synapse using the sharing settings and conditions for use. 
+excerpt: Learn how to set access controls and data governance on Synapse using `Sharing settings` and `Conditions for Use`. 
 ---
```

Comment:
Access Controls seems like a more technical term than share and restrict content?  Also for the summary we shouldn't use the backtick notation as it would visually different in the summary.